lib/iris/fileformats/netcdf.py

```diff
@@ -84,7 +84,7 @@
              'coordinates', '_FillValue', 'formula_terms', 'grid_mapping',
              'leap_month', 'leap_year', 'long_name', 'missing_value',
              'month_lengths', 'scale_factor', 'standard_error_multiplier',
-             'standard_name', 'units', 'valid_max', 'valid_min', 'valid_range']
```

Comment:
> I'm just not convinced that removing these from the _CF_ATTRS list is a good idea... From looking at the cf conventions document it seems this is an appropriate place to keep them...

i disagree with this @dkillick 

the behaviour of these attributes is not controlled by any Iris code

as @hdyson illustrates this is an active blocker to activity, there is no way to write out these attributes from Iris and they are ignored at load time